src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -569,11 +569,19 @@ describe('ReactPropTypes', function() {
 
   describe('OneOf Types', function() {
     it('should fail for invalid argument', function() {
+      spyOn(console, 'error');
+      var warn = console.error;
+      console.error = jest.genMockFn();
```

Comment:
No need to do this, spies are mock functions. And no need to assign to a temporary variable and back - spies are cleaned up automatically. Here's a complete & short example usage: https://github.com/facebook/react/blob/master/src/addons/__tests__/ReactFragment-test.js#L93-L100